test/integration/aws/default/build/rds.tf

```diff
@@ -0,0 +1,25 @@
+# Contains resources and outputs related to testing the aws_rds_* resources.
+
+#======================================================#
+#                    RDS Instances
+#======================================================#
+
+
+resource "aws_db_instance" "default" {
+  allocated_storage    = 20
+  storage_type         = "gp2"
+  engine               = "mysql"
+  engine_version       = "5.6.37"
+  instance_class       = "db.t2.micro"
+  identifier           = "test-instance-id"
```

Comment:
To avoid testing collisions, we need to prefix identifiers.  Typically we use ${terraform.env}.